sign/eddilithium3.go

```diff
@@ -0,0 +1,108 @@
+package sign
+
+import (
+	"github.com/cloudflare/circl/sign/eddilithium3"
+
+	cryptoRand "crypto/rand"
+	"encoding/asn1"
+	"errors"
+)
+
+type edDilithium3Scheme struct{}
```

Comment:
This is a zero-sized data, which is ok. What if this is an integer, which can also be used as identifier in the list of schemes.

Reply:
You can't use a specific integer as a type.